skipper/runner.py

```diff
@@ -53,14 +52,12 @@ def _run_nested(fqdn_image, environment, command, interactive, name, net, volume
     for env in environment:
         docker_cmd += ['-e', env]
 
-    user = getpass.getuser()
-    user_id = os.getuid()
-    docker_cmd += ['-e', 'SKIPPER_USERNAME=%(user)s' % dict(user=user)]
-    docker_cmd += ['-e', 'SKIPPER_UID=%(user_id)s' % dict(user_id=user_id)]
     docker_cmd += ['-e', 'HOME=%(homedir)s' % dict(homedir=homedir)]
 
+    user_id = os.getuid()
     docker_gid = grp.getgrnam('docker').gr_gid
-    docker_cmd += ['-e', 'SKIPPER_DOCKER_GID=%(docker_gid)s' % dict(docker_gid=docker_gid)]
+
+    docker_cmd += ['-u', '%s:%s' % (user_id, docker_gid)]
```

Comment:
as far as i remember, this flag (`-u`) was available when skipper was written. 
the idea was that we wanted the user inside the container to be with the same name the user running skipper (not only the uid). there were some utilities that used the username (maybe those utils are not used anymore).

Reply:
also, i think that running with `-u` assumes that the given `uid` already exists inside the container, and this assumption is not always true (depends on the number of users created so far on the machine where we run skipper).

Reply:
There is no problems with having a uid that does not exists in the container, it creates it.
But I just got a failure of a repo that was expected user to appear in the passwd file...
Any ideas how to handle it? or this entrypoint is the only option?

Reply:
BTW, thanks for the review :-D

Reply:
weak :-)
Closing...